src/Menu/index.jsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import './style.scss';
+import { menuDivider } from '../variables/export-img';
+import SalatList from './SalatList';
+import StarterList from './StartersList';
+import AppetisersList from './AppetisersList';
+import MainList from './MainList';
+import PartOfMenu from '../PartOfMenu';
+
+const Menu = () => (
```

Comment:
export

Reply:
should i export all function in same line woth the funcion initialising